src/router.js

```diff
@@ -14,7 +14,7 @@ router.post('/register', (req, res, next) => {
 
 // login
 
-router.get('/login', (req, res, next) => {
+router.post('/login', (req, res, next) => {
   res.sendStatus(200);
   next();
```

Comment:
I am not sure you need next() function in part 1 as you are not handling any error messages using catch method

Reply:
I agree. However, i am trying to get in the habit of adding next so i don't forget it in my code in the future.